src/app.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="description" content="Svelte demo app" />
+    <link rel="manifest" href="/manifest.json" />
     <link rel="icon" href="%svelte.assets%/favicon.png" />
```

Comment:
suggestion (non-blocking): maybe we could also change this favicon in this PR?) I've made a rounded version without shadows in Figma, you can grab it there if you want to